### PR TITLE
Channel edits - Search for empty tags

### PR DESCRIPTION
**Pull request by jkemp814**
Focusing on empty tvg-country, tvg -language, tvg-logo, and group-title. Continue to search for channels with missing, or non-working logo links.

### CODE REVIEW

**Comment by jkemp814** [comment]
Can someone listen to the channel and identify the language being spoken?

Pluto TV Nick Jr. Club - line 4552

**Comment by Tribalize** [comment]
> Can someone listen to the channel and identify the language being spoken?
> 
> Pluto TV Nick Jr. Club - line 4552

Sounds russian. But i cant confrim

**Comment by Dum4G** [comment]
> Sounds russian. But i cant confrim

Lol. It's Spanish or Portuguese

**Comment by jkemp814** [comment]
I even tried a language identifier, and it couldn't pick it up

File: channels/us.m3u [hunk 1]
```diff
@@ -4549,13 +4549,13 @@ https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5f4796368174910007
...
 #EXTINF:-1 tvg-id="PlutoTVNickJr.us" tvg-name="Pluto TV Nick Jr." tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5ca6748a37b88b269472dad9/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr.
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5ede45451dce190007ef9ff2/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=d1d4cf70-0714-11eb-a59f-0242ac110002&deviceId=5ede45451dce190007ef9ff2&deviceVersion=DNT&appVersion=DNT&deviceDNT=1&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=DE&serverSideAds=false
-#EXTINF:-1 tvg-id="PlutoTVNickJrClub.us" tvg-name="Pluto TV Nick Jr. Club" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5f121460b73ac6000719fbaf/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr. Club
+#EXTINF:-1 tvg-id="PlutoTVNickJrClub.us" tvg-name="Pluto TV Nick Jr. Club" tvg-country="US" tvg-language="" tvg-logo="https://images.pluto.tv/channels/5f121460b73ac6000719fbaf/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr. Club
```
Comment (Dum4G) [inline channels/us.m3u:4552]: I think it isn't wrong that it is Spanish
In October of 2019, a Latino/Spanish version of the channel was launched. Further, in April 2020, a Latin American feed of Nick Jr. Pluto TV was launched. Known as Nick Jr. Club, the network's logo is in the shape of a cloud with "Nick Jr." in it on a light blue background.
https://nickelodeon.fandom.com/wiki/Nick_Pluto_TV#Nick_Jr._Pluto_TV

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/us.m3u:4552]: I know they have a Latino version, but I thought they have Swedish, Russian, Portuguese also. Doesn't sound traditional Spanish. I'm in the Southwest, and it sounds more Russian to me. That's what I took it out until I can confirm what language that is. I will follow up.

Comment (jkemp814) [reply channels/us.m3u:4552]: I can put it Spanish back in, if you all like, until I know for sure.

Comment (Dum4G) [reply channels/us.m3u:4552]: I changed my mind. From the info I found some of Pluto Channels are intended for Brazil and Google Translate speech translation reacts more to Portuguese language

Comment (jkemp814) [reply channels/us.m3u:4552]: So you want it as Portuguese?

**Review by Dum4G — APPROVED** [review]

**Pull request merged**